### ISSUE TITLE
\copydoc stops copying on \section (Origin: bugzilla #627115)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#3903]
status NEW severity *major* in component *general* for *---*
Reported in version *1.7.1* on platform *Other*
Assigned to: Dimitri van Heesch

On 2010-08-17 08:23:47 +0000, Davor wrote:

    In a few post on the net I found people asking why \copydoc doesn't also
    copy \section. For example take this unanswered post by Matthias:
    https://sourceforge.net/mailarchive/forum.php?thread_name=1E47644D-7CAB-
    43EE-A145-487DD9961E9C%40nld.ds.mpg.de&forum_name=doxygen-users
    
    Basically, it copies all until it encounters a section: then it stops.
    
    So people requesting this either use \copydoc in a wrong way, or this is
    indeed a bug. I don't really see a third option.
    
    Kind regards,
    Davor

**Comment by albert-github** [comment]
Besides the code from the question
```
/**\file test.h \brief Test file.

Here is some documentation.

\section firstsec One section
with additional documentation
*/
```

And a small file with:
``` \copydoc test.h```

Tested with version 1.7.1 and didn't see the section part at the `\copydoc`, with version 1.8.14 the section part is present.

I think the problem has been solved in the mean time.
